### PR TITLE
Reuse of OWTF against the samet target and plugin with -f option works

### PR DESCRIPTION
**Pull request by Sentient07**

### CODE REVIEW

File: framework/config/config.py [hunk 1]
```diff
@@ -202,10 +202,15 @@ def LoadProfiles(self, profiles):
     def LoadTargets(self, options):
         scope = self.prepare_url_scope(options['Scope'], options['PluginGroup'])
         added_targets = []
+
         for target in scope:
+            added_targets.append(target)
+            force_option = False
+            if '-f' in options['argv']:
+                cprint("Test")
```
Comment (tunnelshade) [inline framework/config/config.py]: You forgot a print statement?

Comment (Sentient07) [inline framework/config/config.py]: Ouch, had that for testing :P Will remove it and send another one. Sorry !

File: framework/db/target_manager.py [hunk 2]
```diff
@@ -163,11 +175,9 @@ def AddTarget(self, TargetURL, session_id=None):
             target_id = config_obj.id
             self.CreateMissingDBsForURL(TargetURL)
             self.SetTarget(target_id)
+
         else:
-            session_obj = self.db.session.query(
```
Comment (DePierre) [inline framework/db/target_manager.py]: By removing these lines above in a ``if`` statement introduces a bug. If the target URL is already know but the user does not use ``-f``, then OWTF falls in the ``else`` statement where ``target_obj`` does not exist, hence an error ``UnboundLocalError``.

**Comment by DePierre** [comment]
@Sentient07 Could you take a look at my comment and fix it? Also could you revert your PEP8 commit because first, it is not the PEP8 OWTF tries to follow, second because it makes the review harder.

**Comment by DePierre** [comment]
@Sentient07 Are you still working on this? I see no fix for the comment I have made on your commit.

**Comment by delta24** [comment]
@DePierre Can you get me up to speed on this? I reverted the PEP8 commit locally.

**Comment by DePierre** [comment]
@delta24 I'll check that ASAP to remember what was going on and get back to you

**Comment by DePierre** [comment]
@delta24 You'll find the corresponding issues' description at #379 and #378. Let me know if you need more details but IMO the two issues should be enough to get you up to speed.

**Comment by delta24** [comment]
Closing this PR for now.

**Pull request closed without merging**